container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -233,7 +321,7 @@ configuration:
     ip: '"((IP_ADDRESS))"'
     networks.default.dns_record_name: '"((DNS_RECORD_NAME))"'
     networks.default.ip: '"((IP_ADDRESS))"'
-    properties.cf_mysql.mysql.advertise_host: mysql-((KUBE_COMPONENT_INDEX)).mysql-set.((KUBERNETES_NAMESPACE)).svc.((KUBERNETES_CLUSTER_DOMAIN))
+    #properties.cf_mysql.mysql.advertise_host: mysql-((KUBE_COMPONENT_INDEX)).mysql-set.((KUBERNETES_NAMESPACE)).svc.((KUBERNETES_CLUSTER_DOMAIN))
```

Comment:
Why do we have this line commented out?

Reply:
I should remove. It does not seem to be required. 
And now I don't remember how it was handled by the main mysql/pxc bump. 
Was this present in the original main mysql ? And removed in the bump ?

Reply:
Indeed, I removed it from the original move to PXC. https://github.com/SUSE/scf/pull/2390/files#diff-a65976ef35c13467fe140d46c6ca0d3aL2777

Reply:
Ok, will remove completely also.

Reply:
Done in commit c7c018469b9310db9fa6bce378b854347b435e9f